app/uk/gov/hmrc/apiplatformoutboundsoap/models/OutboundSoapMessage.scala

```diff
@@ -22,13 +22,53 @@ import org.joda.time.DateTime
 import java.util.UUID
 import scala.collection.immutable
 
-case class OutboundSoapMessage(globalId: UUID, messageId: Option[String], soapMessage: String, status: SendingStatus, createDateTime: DateTime)
+sealed trait OutboundSoapMessage {
+  val globalId: UUID
+  val messageId: Option[String]
+  val soapMessage: String
+  val status: SendingStatus
```

Comment:
I probably don't need this attribute as now we have separate classes for each status.

Reply:
After reviewing the fallout from doing this, we decided to leave this attribute in place